.github/workflows/release.yml

```
@@ -0,0 +1,166 @@
+          pnpm -C apps/api build
+          pnpm -C apps/web build
+          
+      - name: Create release artifacts
+        run: |
+          mkdir -p release-artifacts
+          
+          # Create API artifact
+          tar -czf release-artifacts/luppa-api-${{ github.ref_name || inputs.version }}.tar.gz \
+            -C apps/api \
+            dist/ package.json README.md
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)